src/util/read-package-json.js

```diff
@@ -0,0 +1,7 @@
+import fs from 'fs-promise';
+import path from 'path';
+
+export default (dir) => {
+  const packageData = fs.readFileSync(path.resolve(dir, 'package.json'), 'utf8');
```

Comment:
In order to use `await` the parent function has to be declared `async`

So this would be

```js
export default async (dir) => {
  const packageData = await fs.readFile(...);
  ...
}
```